service.autosubs/default.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
```

Comment:
entrypoint files are supposed to be small and just import the main codebase. The reason is that this is the only file that is not compiled to bytecode by kodi's python interpreter. Can you do something like:

```python
import sys
from resources.lib import myscript

myscript.run(sys.argv)
```

Move this to `/resources/lib/myscript.py`

Reply:
ok, will do. On separate commit, to keep other changes visible